### ISSUE TITLE
[SYCL] No info::device::atomic64 in SYCL2020 or extensions

### ISSUE THREAD

**Issue by KornevNikita** [intel/llvm#14920]
### Describe the bug

See: https://github.com/intel/llvm/blame/sycl/sycl/source/detail/device_impl.cpp#L375
Looks like when atomic64 aspect was introduced, also info::device::atomic64 was created.
There is no such query in any spec. Should it be deleted?

### To reproduce

#include <sycl/sycl.hpp>

```
int main() {
  sycl::device dev;
  dev.get_info<sycl::info::device::atomic64>();
}
```

clang++ -fsycl main.cpp

### Environment

- OS: [e.g Windows/Linux]
- Target device and vendor: [e.g. Intel GPU]
- DPC++ version: [e.g. commit hash or output of `clang++ --version`]
- Dependencies version: [e.g. the output of `sycl-ls --verbose`]

**Comment by AlexeySachkov** [comment]
Yes, it should be removed, but let's to through a deprecation warning first

**Comment by KornevNikita** [comment]
Note: as I remember there are more redundant info queries than `info::atomic64`, need to find and remove them all.

**Comment by KornevNikita** [comment]
> Yes, it should be removed, but let's to through a deprecation warning first

Was deprecated [here](https://github.com/intel/llvm/pull/15740). Need to remove it at some point:)

**Comment by Robertkq** [comment]
Hi @KornevNikita, I see that it was deprecated a long time ago, almost 2 years. Should it still be removed?
If yes, can I be assigned this issue?

**Comment by KornevNikita** [comment]
@Robertkq hi, assigned to you.
Note - we can't remove it right now, but we can guard all related code with the "#ifndef __INTEL_PREVIEW_BREAKING_CHANGES", so we'll remove this with the next major release.

**Comment by Robertkq** [comment]
Thank you for assigning, happy to work on this :)
Q: How can I find out when is the next major release? I'd like to take my time to understand the underlying code as well, how different `info` is implemented for different properties as well but to have the PR merged in time for next release!

**Comment by KornevNikita** [comment]
We will remove all code guarded with "__INTEL_PREVIEW_BREAKING_CHANGES" in the ~EOY2026 (Nov to Jan 2027).

**Comment by KornevNikita** [comment]
Fixed.
